runtime/doc/treesitter.txt

```diff
@@ -676,16 +676,24 @@ add_directive({name}, {handler}, {force})                    *add_directive()*
 
     Parameters: ~
       • {name}     (string) Name of the directive, without leading #
-      • {handler}  function(match:string, pattern:string, bufnr:number,
-                   predicate:function, metadata:table)
+      • {handler}  function(match:table, pattern:string, bufnr:number,
+                   predicate:string[], metadata:table)
+                   • match: see |treesitter-query|
+                     • node-level data are accessible via `match[capture_id]`
+
+                   • pattern: see |treesitter-query|
+                   • predicate: list of strings containing the full directive
+                     being called, e.g. `(node (#set! conceal "-"))` would get
+                     the predicate `{ "#set!", "conceal", "-" }`
 
 add_predicate({name}, {handler}, {force})                    *add_predicate()*
     Adds a new predicate to be used in queries
 
     Parameters: ~
       • {name}     (string) Name of the predicate, without leading #
-      • {handler}  function(match:string, pattern:string, bufnr:number,
-                   predicate:function)
+      • {handler}  function(match:table, pattern:string, bufnr:number,
+                   predicate:string[])
+                   • see |add_directive()| for argument meanings
```

Comment:
```
test/functional/lua/help_spec.lua:22: invalid tags in :help docs
 *[add_directive()] = 'treesitter.txt' }
```

This is failing because the tag was changed to `vim.treesitter.query.add_directive()` on master. 

Pushing a fix...